mm/Kconfig

```diff
@@ -191,4 +191,8 @@ config MM_BACKTRACE_DEFAULT
 	default n
 	depends on DEBUG_MM
 
+config MM_VM_MAP
+       bool
```

Comment:
Should it be: bool "Virtual Memory Map List Support" in other to let the user to select it? Or is it suppose to only be selected by other drivers?

Reply:
My intention is that it is only selected by other drivers, which use it. After all, it doesn't provide any "feature" by itself do there is no point in selecting it separately. For now, it is primarly needed to provide a generic mechanism for doing munmap in filesystems&drivers - which is not really possible now.